Assets/Package/Runtime/Console/AutocompleteHandler.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System.Collections.Generic;
+
+namespace BroWar.Debugging.Console
+{
+    public class AutocompleteHandler
+    {
+        private IList<string> optionList = new List<string>();
+
+        private string GetShortestValidMatchStarting(string word)
+        {
+            string bestMatch = string.Empty;
+            word = word.ToLower();
+
+            foreach (var option in optionList)
+            {
+                if (option.ToLower().StartsWith(word)
+                   && !option.Equals(word))
+                {
+                    if (string.IsNullOrEmpty(bestMatch) || option.Length < bestMatch.Length)
+                    {
+                        bestMatch = option;
+                    }
+                }
+            }
+
+            return bestMatch;
+        }
+
+        private string GetShortestValidMatchContaining(string word)
+        {
+            string bestMatch = string.Empty;
+            word = word.ToLower();
+
+            foreach (var option in optionList)
+            {
+                if (option.ToLower().Contains(word)
```

Comment:
Tego ifa masz w dwóch miejscach więc dobrze to do osobnej funkcji wrzucić imho

Reply:
Otóż nie, tutaj jest Contains, a w drugim miejscu jest StartsWith.

Reply:
Tu mnie masz. Zcachuj option.ToLower() w takim razie i podaj w obu przypadkach